src/jdk.crypto.ec/share/classes/sun/security/ec/SunEC.java

```diff
@@ -319,9 +318,8 @@ void putEntries() {
         /*
          *  Key Pair Generator engine
          */
-        putService(new ProviderService(this, "KeyPairGenerator",
-            "EC", "sun.security.ec.ECKeyPairGenerator",
-            List.of("EllipticCurve"), ATTRS));
+        putService(new ProviderServiceA(this, "KeyPairGenerator",
+            "EC", "sun.security.ec.ECKeyPairGenerator", ATTRS));
```

Comment:
How about the EllipticCurve alias? Are we dropping it?

Reply:
Hmm, I searched various security docs such as SunEC provider and standard names, there is no mentioning of EllipticCurve alias. So probably ok to drop it.

Reply:
Er, maybe I can add it into KnownOIDs? `EC("1.2.840.10045.2.1", "EC", "EllipticCurve")`.